test/functions/iterators/bharshbarg/recurseHeapBug.chpl

```diff
@@ -1,7 +1,7 @@
 
 record R {
   var x : int;
-  proc R() { }
+  proc init() { }
```

Comment:
How about remove init() altogether? While it does not matter for this test, it will spare future readers the question "why is this initializer here?".